org.ekstep.lessonbrowser-1.4/editor/plugin.js

```diff
@@ -22,14 +22,7 @@ org.ekstep.collectioneditor.basePlugin.extend({
         var templatePath = org.ekstep.contenteditor.api.resolvePluginResource(this.manifest.id, this.manifest.ver, "editor/lessonBrowser.html");
         var controllerPath = org.ekstep.contenteditor.api.resolvePluginResource(this.manifest.id, this.manifest.ver, "editor/lessonBrowserApp.js");
         org.ekstep.contenteditor.api.getService('popup').loadNgModules(templatePath, controllerPath);
-        ecEditor.addEventListener('editor:invoke:viewall', this.initPreview, this)
-       // this.registerRepo(this.getEkstepRepo());
-    },
-
-    registerRepo: function(repo) {
```

Comment:
Have you verified this method in not called by any plugin?

Reply:
No. this method is not calling by any plugin. This was already commented in lessonbrowser-1.3 plugin so after the version bump up I have removed this unused code
-cc @pallakartheekreddy